### DISCUSSION TITLE
Tunneling into devices on a client network

### DISCUSSION THREAD

**Discussion by Sonictech1953** [amidaware/tacticalrmm#1507] [Q&A] [answered]
I'm just getting started with tacticalrmm and it looks pretty nice so far.  One thing I'm missing is some mechanism to build a tunnel to devices that don't/can't have an agent installed (printers, switches, pc's that haven't had one installed yet etc).  How is everyone else dealing with this?  I've seen some mention of a script by dinger1986 but I'm not seeing it in my list of scripts.

**Comment by dinger1986** [comment, marked as answer]
You can use mesh router for this which will be available via your mesh.yourdomain.com

**Reply by Sonictech1953** [reply]
Thanks, that seems to get the job done.
